.github/workflows/buildAndTest.yaml

```diff
@@ -116,7 +116,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven-
       - name: Setup gcloud
-        uses: google-github-actions/setup-gcloud@master
+        uses: google-github-actions/setup-gcloud@main
```

Comment:
This repo appears to still uses `master`, but I'm not sure if GitHub is doing some alias magic on the backend to make this work?

Reply:
Yes, you are right, any idea why?

Reply:
Dunno - but it does look like there may be some [tooling](https://github.com/github/renaming#creating-a-seamless-move-for-existing-repositories-) coming to support automatically renaming

Reply:
Nice feature coming. Thanks for the heads up!